src/orc.cpp

```diff
@@ -67,41 +67,59 @@ std::string_view path_to_symbol(std::string_view path) {
 }
 
 /**************************************************************************************************/
+
```

Comment:
These changes refactor `find_attribute_conflict` to make it easier to understand.